apps/glusterfs/operations_block_volume.go

```diff
@@ -376,6 +376,295 @@ func (bvc *BlockVolumeCreateOperation) CleanDone() error {
 	})
 }
 
+// BlockVolumeExpandOperation implements the operation functions used to
+// expand an existing blockvolume.
+type BlockVolExpandReclaim struct {
```

Comment:
Let's lowercase this name to ensure future readers of the code that's it is private to the app logic.